src/nvim/ex_docmd.c

```diff
@@ -6643,9 +6649,14 @@ static void do_ucmd(exarg_T *eap)
     cmd = USER_CMD_GA(&curbuf->b_ucmds, eap->useridx);
   }
 
+  if (preview) {
+    assert(cmd->uc_preview_luaref > 0);
```

Comment:
we need this check even in release builds

```suggestion
    if (cmd->uc_preview_luaref <= 0) {
        abort():
    }
```

Reply:
Why so? It's logically impossible for this to be true since the `EX_PREVIEW` flag is only true when the Lua function callback is provided. This assert was mainly added for internal functions.